### PR TITLE
Initial integration of meta-demo layer

### PR DESCRIPTION
**Pull request by andr2000**

### CODE REVIEW

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/inc/dom0-core-image.inc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    xen-flask \
+    guests \
+"
+
+populate_append() {
+	install -m 0644 ${DEPLOY_DIR_IMAGE}/xen-${MACHINE}.gz ${DEST}/xen.gz
+}
+
+IMAGE_INSTALL_append = " \
+    guest-images \
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/inc/dom0-core-image.inc:13]: I would eliminate guest-images right here.

**Review by aanisov — APPROVED** [review]
I would rename back meta-rcar-gen3-xen to meta-rcar-gen3.

Anyway:
Reviewed-by: Andrii Anisov andrii_anisov@epam.com

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/inc/dom0-core-image.inc (hunk 1 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/inc/dom0-core-image.inc:13]: will do as a separate patch

Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/inc/dom0-core-image.inc:13]: simply removing guest-images results in:
ERROR: linux-renesas-4.9.0+gitAUTOINC+23445f9f6a-r1 do_package: QA Issue: linux-renesas: Files/directories were installed but not shipped in any package:
  /xen
  /xen/domu.dtb
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
linux-renesas: 2 installed and not shipped files. [installed-vs-shipped]
ERROR: linux-renesas-4.9.0+gitAUTOINC+23445f9f6a-r1 do_package: Fatal QA errors found, failing task.

so, this not only needs guest-images removal, but also some rework of guests.bb

**Comment by lorc** [comment]
Looks fine for me
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**